examples/apps/mouse-drawing/README.md

```diff
@@ -0,0 +1,9 @@
+# Mouse drawing demo
+
+This example shows how to receive mouse and handle mouse events.
+
+To run this demo:
+
+```shell
+cargo run -p mouse-drawing --release
```

Comment:
Mostly because lag matters for this. I doubt it's a problem except on very slow machines though, and I don't have a good test for this, so will remove.